src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -49,26 +61,12 @@ public function __construct(Environment $twig, bool $debug)
      * be used.
      *
      * @return Response
-     *
-     * @throws \InvalidArgumentException When the exception template does not exist
      */
-    public function showAction(Request $request, FlattenException $exception, DebugLoggerInterface $logger = null)
+    public function showAction(Request $request, FlattenException $exception)
     {
-        $currentContent = $this->getAndCleanOutputBuffering($request->headers->get('X-Php-Ob-Level', -1));
-        $showException = $request->attributes->get('showException', $this->debug); // As opposed to an additional parameter, this maintains BC
-
-        $code = $exception->getStatusCode();
-
-        return new Response($this->twig->render(
-            (string) $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
```

Comment:
you are not calling `findTemplate` anymore. So custom controllers overwriting the method will not work as expected anymore. Maybe it's easier to just create a new controller and not touch the old, deprecated one? Then people just need to change the `twig.exception_controller` config to set the new controller.

Reply:
Yes, makes more sense after this https://github.com/symfony/symfony/pull/31398#discussion_r298313966